src/symbolic_memory.ml

```diff
@@ -5,6 +5,8 @@
 module Intf = Interpret_functor_intf
 module Value = Symbolic_value.S
 
+let ( let* ) = Stdlib.Result.bind
```

Comment:
You can use `open Syntax` instead

Reply:
Atm, I cannot use the `let*` defined in syntax as it uses `Owi.Result.t` which is `('a, string) Stdlib.Result.t`. I'm using `('a, Trap.t) Stdlib.Result.t` so that I'm able to raise `Error trap` in the symbolic memory and then `Choice.trap` in `symbolic.ml`

Reply:
Oh OK. I need to change our error type at some point. Could you add a `TODO: use Syntax module` comment so that I think to update it when I change the types ?

Reply:
Yes, can do :smiley: